sktime/annotation/igts.py

```diff
@@ -0,0 +1,486 @@
+# -*- coding: utf-8 -*-
+"""
+Information Gain-based Temporal Segmentation.
+
+Information Gain Temporal Segmentation (IGTS) is a method for segmenting
+multivariate time series based off reducing the entropy in each segment [1]_.
+
+The amount of entropy lost by the segmentations made is called the Information
+Gain (IG). The aim is to find the segmentations that have the maximum information
+gain for any number of segmentations.
+
+References
+----------
+.. [1] Sadri, Amin, Yongli Ren, and Flora D. Salim.
+    "Information gain-based metric for recognizing transitions in human activities.",
+    Pervasive and Mobile Computing, 38, 92-109, (2017).
+    https://www.sciencedirect.com/science/article/abs/pii/S1574119217300081
+
```

Comment:
this should go into class docstring if not there already

Reply:
It is also present in the class docstrings